recipes/proposal/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.32.0"
+
+
+class PROPOSALConan(ConanFile):
+    name = "proposal"
+    homepage = "https://github.com/tudo-astroparticlephysics/PROPOSAL"
+    license = "LGPL-3.0"
+    exports_sources = "CMakeLists.txt"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "monte Carlo based lepton and photon propagator"
+    topics = ("propagator", "lepton", "photon", "stochastic")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_python": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_python": False,
+    }
+
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("cubicinterpolation/0.1.4")
+        self.requires("spdlog/1.8.2")
+        if self.options.with_python:
+            self.requires("pybind11/2.6.2")
+
+    @property
+    def _minimum_compilers_version(self):
+        return {"gcc": "5", "clang": "6", "apple-clang": "6"}
```

Comment:
I'm able to build and consume proposal with clang 6 (with nlohmann_json unvendored, didn't try with vendored lib).

Reply:
Done in e461b8e.